packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/providerValidator.ts

```diff
@@ -10,6 +10,7 @@ import { vsphereProviderValidator } from './vsphere/vsphereProviderValidator';
 
 export function providerValidator(
   provider: V1beta1Provider,
+  subType: 'esxi' | 'vcenter',
   secret: IoK8sApiCoreV1Secret,
 ): ValidationMsg {
   let validationError: ValidationMsg;
```

Comment:
nit suggestion:
Remove let and break statements and use return statements instead

Reply:
I can add a constant for ```'esxi' | 'vcenter'```, but otherwise, all of these suggestions are code that were not introduced in this PR and should be handled by a separate tech debt issue.

Reply:
And actually, that type definition already existed elsewhere so I've imported that and used it here.